src/sbl_filing_api/entities/models/dto.py

```diff
@@ -60,6 +60,7 @@ class ContactInfoDTO(BaseModel):
     hq_address_zip: str
     email: str
     phone_number: str
+    extension: str | None = Field(None, max_length=254)
```

Comment:
What happens when `max_length` is exceeded? I can see from the test that it raises a `422`, which seems appropriate. Does Pydantic give a meaningful error too?

Reply:
You'll get a response json that looks like:
{'error_name': 'Request Validation Failure', 'error_detail': "[{'type': 'string_too_long', 'loc': ('body', 'phone_ext'), 'msg': 'String should have at most 254 characters', 'input': '12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890', 'ctx': {'max_length': 254}}]"}

Reply:
Gotcha. OK, I think that is fine as far as this PR goes. I don't love these big Pydantic barf error messages, but that's an issue for another day. 😄